draft-ietf-masque-connect-udp-listen.md

```diff
@@ -164,6 +166,12 @@ and therefore is to be ignored). This document does not define any parameters
 for the connect-udp-listen header field value, but future documents might define
 parameters. Receivers MUST ignore unknown parameters.
 
+
+# The proxy-public-address Response Header Field {#response}
+Upon accepting the request, the "proxy-public-address" response header,
+MUST contain the IP and the UDP port which are allocated to the client by the
```

Comment:
Do we want to make this mandatory? I was imagining this being optional

Reply:
In what situation do you imagine a proxy would not want to send this? I'd assume that nobody really cares about these ~20 bytes more.